### PR TITLE
added homework for lesson31 - if - loops

### PR DESCRIPTION
**Pull request by siefimov**
please check homework

### CODE REVIEW

File: lesson31-if-loops/index.js [hunk 1]
```diff
@@ -0,0 +1,251 @@
+//* ===== Minimum =====
+/* 1. Запитай у користувача його вік і визначи, ким він є: дитиною (0-11), 
+підлітком (12-17), дорослим (18_59) або пенсіонером (60 ...), передбач 
+можливість введення невірних даних.*/
+const age = prompt("Enter your age");
+
+if (age === null || age === " " || age === "") {
```
Comment (marynabudz1) [inline lesson31-if-loops/index.js:7]: what if undefined or "ff" ?

File: lesson31-if-loops/index.js [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+    default:
+      console.log("unrecognized");
+  }
+}
+
+/* 2. Запитай у користувача число від 0 до 9 і виведи йому спецсимвол, який розташований 
+на цій клавіші (1 !, 2 @, 3 # і т. д).*/
+let number = prompt("Enter any number from 0 to 9");
+
+if (number === null || number === " " || number === "") {
```
Comment (marynabudz1) [inline lesson31-if-loops/index.js:33]: what if undefined or "ff" ?

File: lesson31-if-loops/index.js [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+    case 9:
+      console.log(`( - special character that is located on keyboard key 9`);
+      break;
+    default:
+      console.log("unrecognized");
+  }
+}
+
+/* 3. Підрахуй суму всіх чисел в заданому користувачем діапазоні. */
+const userArray = [20, 40, 60];
```
Comment (marynabudz1) [inline lesson31-if-loops/index.js:74]: user should enter diapason like from 10 to 44
and you need to count each number between 10 and 44

**Review by marynabudz1 — CHANGES_REQUESTED** [review]

File: lesson31-if-loops/index.js (hunk 1 above)
Comment (siefimov) [reply lesson31-if-loops/index.js]: fixed

File: lesson31-if-loops/index.js (hunk 2 above)
Comment (siefimov) [reply lesson31-if-loops/index.js]: fixed

File: lesson31-if-loops/index.js (hunk 3 above)
Comment (siefimov) [reply lesson31-if-loops/index.js:74]: done

**Pull request merged**